config.json

```diff
@@ -71,6 +71,14 @@
         "prerequisites": [],
         "difficulty": 1
       },
+      {
+        "slug": "clock",
+        "name": "Clock",
+        "uuid": "b1ebc201-5d23-4953-88d7-58c82def95b3",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1
```

Comment:
This seems more difficult than a 1 to me

Reply:
Oops, good catch, I meant it to be a 2. Would you even make it a 3?